e2e/ui/input/jobs/service-discovery/service-discovery.js

```diff
@@ -0,0 +1,665 @@
+export const jobSpec = {
```

Comment:
I assume that an awful lot of these properties are the defaults; any way we could make this file something a little easier to reason about if it needed to be modified / duplicated in the future?

Reply:
We totally can. Maybe I can team up with @angrycub and give this a refactor.

Reply:
I cut a [ticket](https://github.com/hashicorp/nomad/issues/14571) to resolve this in a future ticket and will resolve the comment on this PR.